addons/note/models/note.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.tools import html2plaintext
+
+
+class Note(models.Model):
+    _name = 'note.note'
+    _inherit = 'mail.thread'
+    _description = "Note"
+    _order = 'sequence'
+
+    # return the default stage for the uid user
+    def _get_default_stage_id(self):
+        return self.env['note.stage'].search([('user_id', '=', self.env.uid)], limit=1)
+
+    name = fields.Text(compute='_compute_name', string='Note Summary', store=True)
+    user_id = fields.Many2one('res.users', string='Owner', default=lambda self: self.env.uid)
+    memo = fields.Html(string='Note Content')
+    sequence = fields.Integer()
+    stage_id = fields.Many2one('note.stage', compute='_compute_stage_id',
+       inverse='_inverse_stage_id', default=_get_default_stage_id, string='Stage')
+    stage_ids = fields.Many2many('note.stage', 'note_stage_rel',
+       'note_id', 'stage_id', string='Stages of Users')
+    open_active = fields.Boolean(string='Active', track_visibility='onchange', default=True, oldname='open')
```

Comment:
what is meaning of open_active ? can not take only active ? field name should be either is_open or only active.